wizeservices/wordpress-migration-cli
# Create python package

#### What does this PR do?
1. Moves all the code to the main module.
2. Adds a setup.py file to be able to install the module.
3. Adds a binary file after installing the module.
#### Where should the reviewer start?
1. Clone the branch.
2. Run:

```
python3 setup.py install
```
#### How should this be manually tested?
1. Run:
   
   ```
   wordpress-migration-cli
   ```
2. Check if it prints the help.
#### What are the relevant tickets?

Close #15
